java/arcs/android/storage/service/DevToolsProxyImpl.kt

```diff
@@ -0,0 +1,24 @@
+package arcs.android.storage.service
+
+import arcs.android.storage.decodeProxyMessage
+import arcs.android.storage.toProto
+import arcs.core.storage.ProxyMessage
+import arcs.core.util.TaggedLog
+
+class DevToolsProxyImpl : IDevToolsProxy.Stub() {
+    private val log = TaggedLog { "DevToolsProxy" }
+    private val onBindingContextProxyMessageCallbacks = mutableSetOf<IStorageServiceCallback>()
+
+    override fun onBindingContextProxyMessage(proxyMessage: ByteArray) {
```

Comment:
Is this intended to be called from the StorageService? or the DevToolsService? or both?  If only the storage-service, consider removing it from the AIDL interface.

Reply:
Removed from the AIDL interface.